pyanaconda/modules/common/constants/namespaces.py

```diff
@@ -96,6 +96,11 @@
     "Baz"
 )
 
+PAYLOAD_BASE_NAMESPACE = (
+    *PAYLOAD_NAMESPACE,
+    "Base"
```

Comment:
This is inconsistent with our usual approach. Ideally, the base namespace for payloads would be `Payload.Payload` and example of a subnamespace `Payload.Payload.DNF`. Otherwise, you are using a namespace of the main module. It can be confusing and cause problems, because we might need these names  in the future.

I wonder if we should use the same solution as for the `Users` service and rename the `Payload` service to `Payloads` since we plan to eventually support multiple payloads anyway.

Reply:
I go with Payloads